plugin-rust-agent/src/main/kotlin/jetbrains/buildServer/rust/AbstractToolProvider.kt

```diff
@@ -24,7 +30,7 @@ abstract class AbstractToolProvider(
 ): AgentLifeCycleAdapter(), ToolProvider {
     private val LOG = Logger.getInstance(AbstractToolProvider::class.java.name)
     private val VERSION_PATTERN = Regex("^$configName[\\s-]([^\\s]+)", RegexOption.IGNORE_CASE)
-    private val PATH_PATTERN = Regex("^.*$configName(\\.(exe))?$", RegexOption.IGNORE_CASE)
+    private val FILE_NAME_PATTERN = Regex("^\\Q$configName\\E(\\.(exe))?$", RegexOption.IGNORE_CASE)
```

Comment:
Why quote?

Reply:
Just to make sure that `configName` doesn't contain regexp

Reply:
Dropped regepx whatsoever. `lowercase`, `removeSuffix` and `equals` is enough